src/neural/loader.cc

```diff
@@ -126,13 +201,13 @@ Weights LoadWeightsFromFile(const std::string& filename) {
 }
 
 std::string DiscoveryWeightsFile() {
-  const int kMinFileSize = 10000000;  // 10 MB
+  const int kMinFileSize = 5000000;  // 5 MB
```

Comment:
I think that there may be some experimental networks like (32x2), which may compress well.
Maybe reduce that further?.. 1MB?.. What is the compressed size of only policy+value head without residual tower weights?